src/mappings/mappingHandlers.ts

```diff
@@ -9,10 +9,17 @@ import {
   LegacyBridgeSwap,
   Message,
   Transaction,
-  TxStatus
+  TxStatus,
+  NativeTransfer
 } from "../types";
 import {CosmosBlock, CosmosEvent, CosmosMessage, CosmosTransaction,} from "@subql/types-cosmos";
-import {ExecuteContractMsg, DistDelegatorClaimMsg, GovProposalVoteMsg, LegacyBridgeSwapMsg} from "./types";
+import {
+    ExecuteContractMsg,
+    DistDelegatorClaimMsg,
+    GovProposalVoteMsg,
+    LegacyBridgeSwapMsg,
+    NativeTransferMsg
```

Comment:
This should be in the first import statement

Reply:
Got it!